new-docs/puppeteer.browser.md

```diff
@@ -9,7 +9,6 @@
 ```typescript
 export declare class Browser extends EventEmitter 
 ```
-<b>Extends:</b> [EventEmitter](./puppeteer.eventemitter.md)

```

Comment:
@AVGP I think you're running an old version of the doc generator - could you `npm install` again to get up to date? Then this diff goes away :)

Reply:
These issues (contributors using older dep versions) are popping up regularly now. I wonder if we should add Git hooks that re-run `npm install` after `git pull`ing any changes to `package.json`, like we do for v8.dev? https://github.com/v8/v8.dev/blob/dc477272d0aff653ffe5a7acb53246af8f3e7c7b/package.json#L24-L30

Reply:
+1 to this. I don't run `npm i` on every pull and I doubt most people do.